packages/legacy/src/common/constants.ts

```diff
@@ -151,6 +152,26 @@ export const usernameSchema = yup
     excludeEmptyString: true,
   });
 
+export const x509PemSchema = yup
+  .string()
+  .label('PEM encoded X.509 certificate')
```

Comment:
```suggestion
  .label('CA certificate')
```

Reply:
The label can be changed by the component using it.  "PEM encoded X.509" is specifically the format of the certificate that is being checked.  "CA" doesn't mean any particular format or encoding.

Reply:
:+1:

one question, can we have ca certs that are not PEM encoded X.509 ?
if we do, we need to allow this types of ca too